### ISSUE TITLE
Drippy Loading Screen - Red Screen, Loading Top Left.

### ISSUE THREAD

**Issue by CandidNebula** [Keksuccino/Drippy-Loading-Screen#73]
**Describe the bug**
Drippy loading screen is fine in edit,
However, it loads top left corner, while the rest of the screen is red. Reloaded game, same issue.
Updated Fancy, no change. Drippy didn't have any updates.
Updated Modrinth beforehand, but I don't know how that could have an effect?

Everything was fine last load, maybe 3-6 months ago. Just came back to this.
No updates since then.
I'm trying to create a modpack, so that's why I'm hopeful it's an easy fix.

**To Reproduce**
Steps to reproduce the behavior (e.g.):
1.Open Modrinth
2. Update Modrinth
3. Open game

I also tried re opening the Drippy editor to change some stuff around and re-save, no change
1. Opened editor
2. Changed image, and back
3. Changed and updated loading bar
4. Save, Exit, Reload
5. No change.

**Expected behavior**
Clear loading screen, Loading bar working, etc.

**Game Log**
https://gist.github.com/CandidNebula/94aae911661657ece2c0a20b57a3906a

**Screenshots**
![screenshotofloadingscreen](https://github.com/Keksuccino/Drippy-Loading-Screen/assets/170223094/2e2e14eb-9640-4ced-9609-c4a770d88351)

**Basic Informations (please complete the following information):**
 - Windows/PC
 - Drippy Version 3.0.2-1.20.1-fabric
 - Fabric 0.92.0+1.20.1 - Loader 0.15.7
 - Minecraft Version 1.20.1
(I have this version due to the mods I want to keep.)

**Active Mods**
https://gist.github.com/CandidNebula/f664db50406da5f0df3470ee5baa8368

**Notes**
It's been weird so far, I went through your incompatible mod list, and nothing jumped out at me. 
I also don't have 'EnhancedVisuals'.
Watch, there's probably a setting I have to turn on/off. lol

Hopefully this helps others looking for the same issue.
- Candid

**Comment by CandidNebula** [comment]
So I figured out what it was! I had recently changed my screen res from 1080px x 1920px to 2560 x 1440. For some reason the mod is confused but will run fine if you switch it back. The question is though, is how do I fix it?

**Comment by GeepFeet** [comment]
Hey did you ever find a solution for this issue. I find that this only happens to me when my gui scale is set to 5 which happens to be the scale that I want to use.

**Comment by CandidNebula** [comment]
> Hey did you ever find a solution for this issue. I find that this only happens to me when my gui scale is set to 5 which happens to be the scale that I want to use.

I did actually! It was a while ago but I want to say that I changed my resolution back to 1080, or updated it in the launcher and it fixed it. Via the mod launcher, I changed my 'launch game resolution' to the resolution it would work at, and it fixed the loading issue. Very strange. I'll see if I can find a picture!

**Comment by Keksuccino** [comment]
Yeah sorry for never replying. You posted that during a pretty rough time for me, so I never really answered, but I also don't really have an explaination for why that happens tbh. It should correctly scale the loading screen no matter the launch window size. It could be a mod conflict, because it works fine for me.
